libcontainer/selinux/selinux.go

```diff
@@ -158,7 +158,9 @@ func Setfilecon(path string, scon string) error {
 // Getfilecon returns the SELinux label for this path or returns an error.
 func Getfilecon(path string) (string, error) {
 	con, err := system.Lgetxattr(path, xattrNameSelinux)
-
+	if err != nil {
+		return "", err
+	}
```

Comment:
We use this function when debugging code, in the past.  Do we really need to remove it just because no one is using it currently?